src/EditorFeatures/TestUtilities/CodeActions/CSharpCodeFixVerifier`2.cs

```diff
@@ -72,5 +74,48 @@ public static async Task VerifyCodeFixAsync(string source, DiagnosticResult[] ex
             test.ExpectedDiagnostics.AddRange(expected);
             await test.RunAsync();
         }
+
+        public static async Task VerifyCodeFixAsync((string first, string second) sources, (string first, string second) fixedSources, int numberOfFixAllIterations)
+            => await VerifyCodeFixAsync(sources, fixedSources, DiagnosticResult.EmptyDiagnosticResults, numberOfFixAllIterations);
+
+        public static async Task VerifyCodeFixAsync((string first, string second) sources, (string first, string second) fixedSources, DiagnosticResult[] expected, int numberOfFixAllIterations)
+        {
+            var test = new Test
+            {
+                TestState = { Sources = { sources.first, sources.second } },
+                FixedState = { Sources = { fixedSources.first, fixedSources.second } },
+                NumberOfFixAllInDocumentIterations = numberOfFixAllIterations
```

Comment:
❗️ `NumberOfFixAllInDocumentIterations` is not the same as `NumberOfFixAllIterations`. We should not confuse the naming.

Reply:
Sure, I can rename the parameter.